### ISSUE TITLE
Training is not Getting Started

### ISSUE THREAD

**Issue by ChaitanyaGaj** [ultralytics/ultralytics#7253]
### Search before asking

- [X] I have searched the YOLOv8 [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

Unable to train YoloV8 on Custom Object detection while executing the command. Its getting struck before Epochs start
!yolo task=detect mode=train model=yolov8s.pt data= data.yaml epochs=1 imgsz=490 plots=True

### Additional

Facing error:
----------
Plotting labels to runs/detect/train2/labels.jpg... 
optimizer: 'optimizer=auto' found, ignoring 'lr0=0.01' and 'momentum=0.937' and determining best 'optimizer', 'lr0' and 'momentum' automatically... 
optimizer: AdamW(lr=0.002, momentum=0.9) with parameter groups 57 weight(decay=0.0), 64 weight(decay=0.0005), 63 bias(decay=0.0)
10 epochs...
Closing dataloader mosaic

      Epoch    GPU_mem   box_loss   cls_loss   dfl_loss  Instances       Size
  0%|          | 0/1 [[00:00](https://www.youtube.com/watch?v=iy34dSwfEsY)<?, ?it/s]

**Comment by glenn-jocher** [comment]
@ChaitanyaGaj hello! It seems like you're encountering an issue where training doesn't proceed past the initial setup phase. The message about plotting labels and setting up the optimizer indicates that the setup steps are working, but the training stalls before the first epoch begins.

This could be due to several reasons, such as a configuration issue, a problem with the dataset, or insufficient system resources. Here are a few steps you can take to troubleshoot the problem:

1. **Check Dataset**: Ensure your `data.yaml` is correctly formatted and the paths to your images and labels are correct.
2. **System Resources**: Verify that your system has enough memory and that the GPU is properly configured and accessible.
3. **Configuration**: Look over your training configuration to make sure all parameters are set appropriately for your custom dataset.
4. **Logs**: Review any logs or error messages that might provide more insight into where the process is getting stuck.

If the issue persists, please provide more details about your setup, including the full error message if there is one, and any additional information about your custom dataset and system environment. This will help us better understand the problem and assist you further. Remember, the community and documentation at Ultralytics Docs are great resources to help you get past these initial hurdles. 🚀
